doc/changelog.rst

```diff
@@ -6,12 +6,90 @@ Changelog
 Motor 3.0
```

Comment:
Are there any notable _new_ features to document?

Reply:
Added applicable "notable improvements" from PyMongo 4.0 and 4.1